Resources/public/js/views/services/ez-locationviewviewservice.js

```diff
@@ -171,6 +172,7 @@ YUI.add('ez-locationviewviewservice', function (Y) {
                     5
                 );
 
+                that.fire('moveContent', {});
```

Comment:
the event should maybe be called movedContent to reflect the fact it's fired after the operation.